apps/dashboard/src/components/workflow-editor/step-utils.ts

```diff
@@ -58,10 +67,25 @@ export const updateStepInWorkflow = (
   };
 };
 
-export const createStep = (type: StepTypeEnum): Step => ({
-  name: STEP_TYPE_LABELS[type] + ' Step',
-  stepId: '',
-  slug: '_st_',
-  type,
-  _id: crypto.randomUUID(),
-});
+export const createStep = (type: StepTypeEnum): StepCreateDto => {
```

Comment:
we used the wrong type here, this is way we passed redundant data to the API such as slug prefix